lib/index.coffee

```diff
@@ -55,4 +55,38 @@ class DockerGC
 			console.log('GC: Done.')
 			return _.every(results)
 
+	getDaemonFreeSpace: () ->
+		# Ensure the image is available (if it is this is essentially a no-op)
+		@docker.pull('alpine')
+		.then (stream) ->
+			new Promise (resolve, reject) ->
+				stream.resume()
+				stream.once('error', reject)
+				stream.once('end', resolve)
+		.then =>
+			@docker.run 'alpine',
+				[
+					'/bin/sh',
+					'-c',
```

Comment:
I don't think commas are necessary here

Reply:
Ah yeah, that's the problem with switching between TS and CS so often...